### PR TITLE
products/anolis23: supports Anolis OS 23

### PR DESCRIPTION
**Pull request by rain-Qing**
Signed-off-by: YuQing [yyq0391@163.com](mailto:yyq0391@163.com)
Signed-off-by: YiLin.Li [YiLin.Li@linux.alibaba.com](mailto:YiLin.Li@linux.alibaba.com)

**Description:**
products/anolis8: supports Anolis OS 23

**Rationale:**

- OpenSCAP now supports many products. Anolis OS 23 ( https://openanolis.cn/ ) is an upstream native community-oriented independent selection based on the theory of operating system hierarchy classification by the OpenAnolis community; natively building a full-stack hardware and software collaborative next generation OS version.So I want to add the produce/anolis23 into OpenSCAP.

- Fixes https://github.com/ComplianceAsCode/content/issues/10538

### CODE REVIEW

**Comment by Mab879** [comment]
Please take a look at the CI failures. The `utils/fix_rules.py` should be in fixing the error reported by CI.

**Comment by rain-Qing** [comment]
> Please take a look at the CI failures. The `utils/fix_rules.py` should be in fixing the error reported by CI.

Thanks for your advice, I have repaired the code again and submitted it. The CI test has been passed.

**Comment by rain-Qing** [comment]
@Mab879 I have resolved the conflict and resubmitted.

Please review it again. Thank you.

File: products/anolis23/transforms/constants.xslt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:include href="../../shared/transforms/shared_constants.xslt"/>
```
Comment (Mab879) [inline products/anolis23/transforms/constants.xslt:3]:
```suggestion
<xsl:include href="../../../shared/transforms/shared_constants.xslt"/>
```

File: products/anolis23/transforms/table-style.xslt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:import href="../../shared/transforms/shared_table-style.xslt"/>
```
Comment (Mab879) [inline products/anolis23/transforms/table-style.xslt:3]:
```suggestion
<xsl:import href="../../../shared/transforms/shared_table-style.xslt"/>
```

File: products/anolis23/transforms/xccdf-apply-overlay-stig.xslt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```
Comment (Mab879) [inline products/anolis23/transforms/xccdf-apply-overlay-stig.xslt:1]: Unless this product has  STIG please remove this file.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR! Please take look at my comments.

**Comment by rain-Qing** [comment]
@Mab879 
Thank you for your comments.  I have processed and pushed again.
Please review it again. Thank you.

File: products/anolis23/transforms/xccdf2table-cce.xslt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:cce="http://cce.mitre.org" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cdf="http://checklists.nist.gov/xccdf/1.1" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+
+<xsl:import href="../../shared/transforms/shared_xccdf2table-cce.xslt"/>
```
Comment (Mab879) [inline products/anolis23/transforms/xccdf2table-cce.xslt:4]:
```suggestion
<xsl:import href="../../../shared/transforms/shared_xccdf2table-cce.xslt"/>
```

File: products/anolis23/transforms/xccdf2table-profileccirefs.xslt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cdf="http://checklists.nist.gov/xccdf/1.1" xmlns:cci="https://public.cyber.mil/stigs/cci" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:ovalns="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+
+<xsl:import href="../../shared/transforms/shared_xccdf2table-profileccirefs.xslt"/>
```
Comment (Mab879) [inline products/anolis23/transforms/xccdf2table-profileccirefs.xslt:4]:
```suggestion
<xsl:import href="../../../shared/transforms/shared_xccdf2table-profileccirefs.xslt"/>
```

**Review by Mab879 — CHANGES_REQUESTED** [review]
Looks like a couple paths are still wrong.

Once these are fixed this should be ready to merge.

**Review by Mab879 — APPROVED** [review]

**Comment by rain-Qing** [comment]
@Mab879 
Thank you for your review and comments.
Now I see that the CI/CD test has passed. (links and screenshots of some of the test results are below)
Is there anything else I need to do before the merger?

[test](https://github.com/rain-Qing/content/actions/runs/5472026122/jobs/9963746209)
image

**Pull request merged**